goldmane/proto/api.proto

```diff
@@ -200,6 +202,12 @@ message Filter {
 
   // Policies matches on policy fields. Combined using logical OR.
   repeated PolicyMatch policies = 8;
+
+  // Reporter filters on the reporter field. Combined using logical OR.
```

Comment:
```suggestion
  // Reporter filters on the reporter field.
```

^ This isn't a repeated field, so there's nothing to combine. That said, perhaps it _should_ be a repeated field, in case we add more reporters in the future and we want to select on multiple at once?

Reply:
Done :+1: